### PR TITLE
Mkampen/add s1 mosaic

### PR DESCRIPTION
**Pull request by Kampen1988**
Add Sentinel-1 monthly mosaics descriptions

### CODE REVIEW

**Comment by Kampen1988** [comment]
Hi @Pratichhya, would you be able to troubleshoot this pull request with me? I have added all the information needed for the Sentinel-1 monthly mosaics, but something is probably missing either in the `collections.json` or the `_render_collections.qmd` script. Feel free to send a meeting invite for today.

**Review by Pratichhya — APPROVED** [review]

File: Data/Datafunctions/data_availability.py [hunk 1]
```diff
@@ -241,11 +241,23 @@ def general(c):
...
+                S3path = c['summaries']['DataAvailability'][i]['S3 Path']
+            except Exception:
+                S3path = ''
         
             if offered_type != '':
-                t.append([offered_type, status, access, spatial, temporal, available_from])
+                t.append([offered_type, status, access, spatial, temporal, available_from, mosaic, polarization, S3path,])
                 note += footnotes
-                headers = ["Timeliness","Archive Status", "Access Type", "Spatial Extent", "Temporal Extent", "Available in Ecosystem from"]
+                headers = ["Timeliness", "Archive Status", "Access Type", "Spatial Extent", "Temporal Extent", "Available in Ecosystem from", "Mosaic", "Polarization", "S3 Path", ]
```
Comment (Pratichhya) [inline Data/Datafunctions/data_availability.py:260]: It seems to work as expected.

However @Kampen1988 @azlinszkysinergise these cells "mosaic, polarization" though added, isn't reflected in the table.

**Comment by Pratichhya** [comment]
content is confirmed by @azlinszkysinergise

File: Data/Datafunctions/data_availability.py (hunk 1 above)
Comment (Kampen1988) [reply Data/Datafunctions/data_availability.py:260]: @Pratichhya , who can I follow up with on the missing table columns and cells? Kaner?

Comment (Pratichhya) [reply Data/Datafunctions/data_availability.py:260]: sure 
cc @KanerLev

Comment (Kampen1988) [reply Data/Datafunctions/data_availability.py:260]: Hi @KanerLev ,
would you be available to provide some guidance on the necessary changes to `data_availability.py` if we want the currently missing columns and cells to be rendered?

Comment (Kampen1988) [reply Data/Datafunctions/data_availability.py:260]: @Pratichhya, is there anyone else who could help me out with this?

Comment (KanerLev) [reply Data/Datafunctions/data_availability.py:260]: Hi @Kampen1988, Pratichhya is not available. 
Yes, I'll be able to help as much as I can. I couldn't write earlier.
Sent you a meeting request, feel free to adjust.

Comment (Kampen1988) [reply Data/Datafunctions/data_availability.py:260]: Solved

**Pull request merged**